### PR TITLE
Update profile photo and project role

### PR DESCRIPTION
**Pull request by songgthu**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Trinh Hoai Song Thu (Sarah)
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/sarah.png" width="200px">
```
Comment (sudarshan2401) [inline docs/AboutUs.md]: I think the name of the file needs to match your github username. 
See: "The filename of the profile photo should be docs/images/github_username_in_lower_case.png" from the 2103 website

Comment (songgthu) [reply docs/AboutUs.md]: oh thank you!

**Review by DonovanJJ — APPROVED** [review]
Looks good from here!

**Review by sudarshan2401 — APPROVED** [review]
LGTM!

**Pull request merged**